### ISSUE TITLE
[DTM0][Spike] Client waits on connections from servers

### ISSUE THREAD

**Issue by cortx-admin** [Seagate/cortx-prvsnr#3496]
As a part of EOS-17644, we introduced a very simple way to ensure that the "servers > client" connections are established: we use sleep() + m0ham. It is enough for a demo, both this not a "debug"-friendly approach: we spawn the client, send HA msg and then we block on the client.

We need to figure out if is possible to get rid of that sleep (and ideally – to get rid of this extra call to m0ham).

DoD:
 * Mandatory: the sleep() goes away.
 * Optional: the extra call to m0ham goes away too.

NOTE: If there is no easy solution for that problem, then we will close this ticket as "WONTFIX" –  HA/Hare support plus the work on the start/stop procedure will wear off this problem.
